src/test/java/com/puppycrawl/tools/checkstyle/grammar/comments/AllBlockCommentsTest.java

```diff
@@ -48,14 +44,15 @@ protected String getPackageLocation() {
 
     @Test
     public void testAllBlockComments() throws Exception {
-        final DefaultConfiguration checkConfig =
-                createModuleConfig(BlockCommentListenerCheck.class);
         final String path = getPath("InputFullOfBlockComments.java");
         lineSeparator = CheckUtil.getLineSeparatorForFile(path, StandardCharsets.UTF_8);
-        execute(checkConfig, path);
-        assertWithMessage("All comments should be empty")
-            .that(ALL_COMMENTS)
-            .isEmpty();
+        final String[] expected = {
+            "8:13: " + "violation found in comment: violation1",
```

Comment:
Please extract the base message to a `private static final` field, like `private static final String VIOLATION_MESSAGE = "violation found in comment:";` and reuse it in all occurrences.

Reply:
@nrmancuso done